toolkit/scripts/cherry_pick.sh

```diff
@@ -0,0 +1,88 @@
+#!/bin/bash
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+set -e
+
+function help {
+    echo "Cherry-pick commit to a specific branch and create Github PR"
+    echo "Usage:"
+    echo "[MANDATORY] -r REPOSITORY -> name of the repository in the format <owner>/<repo-name>"
+    echo "[MANDATORY] -p PR_NUMBER -> number of the PR to cherry-pick commit from"
+    echo "[MANDATORY] -t TARGET_BRANCH -> target branch to cherry-pick commit to"
+}
+
+function collect_pr_info {
+    repo=$1
+    pr_number=$2
+
+    commit_hash=$(gh pr view $pr_number --repo $repo --json mergeCommit --jq '.mergeCommit.oid')
+    original_pr_title=$(gh pr view $pr_number --repo $repo --json title --jq '.title')
+}
+
+function cherry_pick {
+    commit_hash=$1
+    target_branch=$2
+    repo=$3
+    pr_number=$4
+    original_pr_title=$5
+
+    tmp_branch="cherry-pick-$target_branch-$commit_hash"
+
+    echo "Cherry picking commit ($commit_hash) to target branch $target_branch"
```

Comment:
For the sake of consistency:
```suggestion
    echo "Cherry picking commit ($commit_hash) to target branch ($target_branch)"
```

And in general I'm personally a fan of surrounding string-like variables with brackets or quotes, because that helps during debugging if for example one variable by accident gets set to an empty string.